docs/getting-started/installation.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Installation
+description: ⚡ A simple guide to installing Spicetify on any platform.
+---
+
+:::tip
+Please read our [**post-installation**](./usage.md#post-installation) guide after installing for information on Marketplace and basic setup.
+:::
```

Comment:
```suggestion
Spicetify is a multiplatform command-line tool to customize the official Spotify client.

## Installation

### Windows

This is the installation method what we recommend for most users. It is the fastest and most reliable way to install Spicetify. 

#### Powershell (pre-built binary)

```powershell
iwr -useb https://raw.githubusercontent.com/spicetify/spicetify-cli/master/install.ps1 | iex
```

Also run the following if you would like to install the [**Spicetify Marketplace**](https://github.com/spicetify/spicetify-marketplace), which gives you access to a tab in Spotify's sidebar that allows you to search for and install _themes_, _extensions_, and _snippets_.
```powershell
iwr -useb https://raw.githubusercontent.com/spicetify/spicetify-marketplace/main/resources/install.ps1 | iex
```

### Linux and MacOS

#### Shell (pre-built binary)
Spicetify CLI
```sh
curl -fsSL https://raw.githubusercontent.com/spicetify/spicetify-cli/master/install.sh | sh
```
Spicetify Marketplace
```sh
curl -fsSL https://raw.githubusercontent.com/spicetify/spicetify-marketplace/main/resources/install.sh | sh
```

<hr/>

## Basic Usage

After installing Spicetify and Spicetify's Marketplace, you can use it to customize your Spotify client using all the available **extensions** and **themes** found in the Marketplace.

### Updating

**_Spotify_**, every now and then, **updates** its client. Since we have no power over this process, you will likely need to **re-apply Spicetify**.

However, the update might have major changes to the client, which means you will need to run `spicetify upgrade` every time you update Spotify. If no update for Spicetify is available, it means that it either still works by simply running `spicetify backup apply`, or that we are still **working on updating Spicetify** to work on the new version.

After running `spicetify upgrade`, you will be prompted to run `spicetify restore backup apply` to have everything back where it was.

:::tip
Please read our [**post-installation**](./usage.md#post-installation) guide after installing for information on Marketplace and basic setup.
:::
```